docs/guidelines/users/global-leaderboard-and-achievement-hunting-rules.md

```diff
@@ -217,6 +217,6 @@ In cases where there is any more complex scripting than repeating the same butto
 
 ## Special Thanks
 
-This document is heavily based on the [100Pals Unified Achievement Hunting Rules](https://100pals.co/rules). This version is trimmed down and tailored with topics that are relevant to the RetroAchievements context.
+This document is heavily based on the [100Pals Unified Achievement Hunting Rules](https://achievementhunting.com/rules) (the "UAHR"). This version is trimmed down and tailored to topics that are specifically relevant to the unique RetroAchievements context, although additional details may be gleaned from the UAHR when needed.
```

Comment:
```suggestion
This document is heavily based on the [100Pals Unified Achievement Hunting Rules](https://achievementhunting.com/rules) (the "UAHR"). This version is trimmed down and tailored to topics that are specifically relevant to the RetroAchievements context.
```
Minor adjustments. We don't want users pointing to an off-site document trying to find loopholes when they're untracked.